app/requests.py

```diff
@@ -0,0 +1,18 @@
+import requests
+from flask import current_app
+import os
+import json
+from dotenv import load_dotenv
+
+load_dotenv()
+
+def use_header(text):
```

Comment:
I like this helper, I would name it something different though since it's not clear it actually makes a request based on its name.

Maybe something like:
```suggestion
def slack_notify(text):
```